scripts/run_yamlcheck.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash -xe
+
+arch=$(uname -m)
+if [ "$arch" = "x86_64" ];
```

Comment:
shoulodn't we skip also for x86/x86_32?

Reply:
do we have slaves that work on x86/x86_32?

Reply:
do we have any slaves that work on x86/x86_32?

Reply:
change it to be on the safe side

Reply:
I think they only support 64:
https://hub.docker.com/r/garethr/kubeval/tags